File: SDJ3/Java/Session7/src/main/java/Person.java


import java.io.Serializable;

public class Person implements Serializable {
    private String name;
    private String Position;
    private int age;
    private boolean  happy;

    public Person(String name, String position, int age, boolean happy) {
        this.name = name;
        Position = position;
        this.age = age;
        this.happy = happy;
    }

    public String getName() {
        return name;
    }

    public String getPosition() {
        return Position;
    }

    public int getAge() {
        return age;
    }

    public boolean isHappy() {
        return happy;
    }

    public void setName(String name) {
        this.name = name;
    }

    public void setPosition(String position) {
        Position = position;
    }

    public void setAge(int age) {
        this.age = age;
    }

    public void setHappy(boolean happy) {
        this.happy = happy;
    }
}

File: DES/session1/Session6 ZZ SDJ3/Person.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Session6_ZZ_SDJ3
{
    class Person
    {
        private string name { get; set; }
        private string Position { get; set; }
        private int age { get; set; }
        private bool happy { get; set; }

        public Person(string name, string position, int age, bool happy)
        {
            this.name = name;
            Position = position;
            this.age = age;
            this.happy = happy;
        }
    }
}
